Test Case 1 Input:
Test Case Input:-112345678910
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 2 Input:
"1""2""3""4""5"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 3 Input:
"1" "2" "3""4" "5" "6""7" "8" "9""3""1" "2" "3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 4 Input:
"10""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 5 Input:
"10""2.5"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 6 Input:
"10""3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 7 Input:
"10" (String)"2" (String)
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 8 Input:
"10A0101010"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 9 Input:
"10a01010"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 10 Input:
"10a0101010"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 11 Input:
"12345"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 12 Input:
"1a0101010"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 13 Input:
"3123"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 14 Input:
"32-202"Test Case Input:"-24-305"Test Case Input:"53-2042"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 15 Input:
"3""1""2""3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 16 Input:
"3" "2""-2""0""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 17 Input:
"3" 2 -2 0 2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 18 Input:
"3" (String)"2" (String)"-2" (String)"0" (String)"2" (String)
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 19 Input:
"3" (String)123
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 20 Input:
"3" (String)2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 21 Input:
"3" 2 -2 0 2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 22 Input:
"3.5""2""-2""0""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 23 Input:
"3\n2\n-2\n0\n2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 24 Input:
"54 2 7 1 -9"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 25 Input:
"54 2 7 1 9"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 26 Input:
"5""10""3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 27 Input:
"5""2""11001"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 28 Input:
"5""2""11a01"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 29 Input:
"5""2""3""4""5""6"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 30 Input:
"5""2""abc123"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 31 Input:
"5""3""-2""0""4""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 32 Input:
"5""4" "2" "7" "1" "9"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 33 Input:
"5" "3" "-2" "0" "4" "2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 34 Input:
"5" (input value is a string instead of an integer)"2""11001"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 35 Input:
"5" (string)"2" (string)"11001" (string)
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 36 Input:
"5.5""3""11001"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 37 Input:
"5\n4 2 7 1"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 38 Input:
"ABC123"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 39 Input:
"Hello World"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 40 Input:
"Hello"103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 41 Input:
"a""b""c"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 42 Input:
"a""b""c""d""e"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 43 Input:
"a"123
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 44 Input:
"a"103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 45 Input:
"abc"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 46 Input:
"abc""10""3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 47 Input:
"abc""10"3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 48 Input:
"abc"011001
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 49 Input:
"abc"123
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 50 Input:
"abc"12345678910
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 51 Input:
"abc"103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 52 Input:
"abc"2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 53 Input:
"abc"2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 54 Input:
"abc"20
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 55 Input:
"abc"21-305
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 56 Input:
"abc"211001
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 57 Input:
"abc"4 2 7 1 9
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 58 Input:
"abc" 2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 59 Input:
"abc" 4 2 7 1 9
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 60 Input:
"abc123"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 61 Input:
"xyz"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 62 Input:
-1
Output:
0
Test Case 63 Input:
-1-1-1-1-1-1-1-1-1-1-1
Output:
0
Test Case 64 Input:
-1012345678910
Output:
0
Test Case 65 Input:
-102345678910
Output:
0
Test Case 66 Input:
-112345678910
Output:
0
Test Case 67 Input:
-11012345678910
Output:
0
Test Case 68 Input:
-12-305
Output:
0
Test Case 69 Input:
-13579111315171921
Output:
0
Test Case 70 Input:
-10
Output:
-10
Test Case 71 Input:
-10Test Case Input:51 0Test Case Input:31 nullTest Case Input:41 22 33 45 6Test Case Input:21 23 4
Output:
-10
Test Case 72 Input:
-100
Output:
-10
Test Case 73 Input:
-1012345678910
Output:
-10
Test Case 74 Input:
-102
Output:
-10
Test Case 75 Input:
-102030
Output:
-10
Test Case 76 Input:
-10.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 77 Input:
-100
Output:
-100
Test Case 78 Input:
-10012345678910
Output:
-100
Test Case 79 Input:
-100.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 80 Input:
-100000000012345678910
Output:
-2000000000
Test Case 81 Input:
-100000000112345678910
Output:
-2000000000
Test Case 82 Input:
-100000000212345678910
Output:
-2000000000
Test Case 83 Input:
-10a0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 84 Input:
-1a0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 85 Input:
-20-305
Output:
0
Test Case 86 Input:
-212345678910
Output:
0
Test Case 87 Input:
-24-305
Output:
0
Test Case 88 Input:
-24005
Output:
0
Test Case 89 Input:
-3
Output:
0
Test Case 90 Input:
-30-252
Output:
0
Test Case 91 Input:
-301-23
Output:
0
Test Case 92 Input:
-302-24
Output:
0
Test Case 93 Input:
-3123
Output:
0
Test Case 94 Input:
-31 22 3
Output:
0
Test Case 95 Input:
-34 2 -7
Output:
0
Test Case 96 Input:
-3.14
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 97 Input:
-5
Output:
-5
Test Case 98 Input:
-5Test Case Input:101 a2 b3 c4 d5 e6 f7 g8 h9 i10 jTest Case Input:31 22 3Test Case Input:410 2020 3030 4040 50Test Case Input:51 22 33 44 55 6
Output:
-5
Test Case 99 Input:
-5-2035710152025
Output:
-5
Test Case 100 Input:
-51234
Output:
-5
Test Case 101 Input:
-512345678910
Output:
-5
Test Case 102 Input:
-51 21 33 43 5
Output:
-5
Test Case 103 Input:
-5103
Output:
-5
Test Case 104 Input:
-52
Output:
-5
Test Case 105 Input:
-52abc123
Output:
-5
Test Case 106 Input:
-52null
Output:
-5
Test Case 107 Input:
-52 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
-5
Test Case 108 Input:
-53
Output:
-5
Test Case 109 Input:
-54 2 7 1 9
Output:
-5
Test Case 110 Input:
-5.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 111 Input:
-500
Output:
-1000
Test Case 112 Input:
0
Output:
0
Test Case 113 Input:
0Test Case Input:31 22 3Test Case Input:41 22 33 4Test Case Input:51 21 33 43 5Test Case Input:61 21 32 42 54 6
Output:
0
Test Case 114 Input:
0Test Case Input:31 22 33 4Test Case Input:51 22 33 44 55 6
Output:
0
Test Case 115 Input:
000
Output:
0
Test Case 116 Input:
00000
Output:
0
Test Case 117 Input:
0123
Output:
0
Test Case 118 Input:
012345678910
Output:
0
Test Case 119 Input:
01 22 33 44 5
Output:
0
Test Case 120 Input:
01 22 33 44 5Test Case Input:31 22 33 44 5Test Case Input:51 22 33 44 55 6Test Case Input:61 22 33 44 55 66 7
Output:
0
Test Case 121 Input:
01 2 3 4 5
Output:
0
Test Case 122 Input:
01 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 251 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 251 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
Output:
0
Test Case 123 Input:
01 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 124 Input:
02468
Output:
0
Test Case 125 Input:
02 33 44 55 6
Output:
0
Test Case 126 Input:
02 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 127 Input:
03Test Case Input:501234Test Case Input:33123Test Case Input:3212-3Test Case Input:32123
Output:
0
Test Case 128 Input:
03 5 2
Output:
0
Test Case 129 Input:
0Test Case Input:34 2 -7
Output:
0
Test Case 130 Input:
0 0 00 0 00 0 031 2 3
Output:
0
Test Case 131 Input:
0 11 22 33 44 5
Output:
0
Test Case 132 Input:
010101
Output:
20100
Test Case 133 Input:
0101010101
Output:
202020100
Test Case 134 Input:
0101010101010101010
Error:
Exception in thread "main" java.util.InputMismatchException: For input string: "0101010101010101010"
	at java.base/java.util.Scanner.nextInt(Scanner.java:2273)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 135 Input:
011011
Output:
22010
Test Case 136 Input:
0a1b0c1d0e1f
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 137 Input:
10-1
Output:
0
Test Case 138 Input:
11
Output:
0
Test Case 139 Input:
11-503
Output:
0
Test Case 140 Input:
110
Output:
0
Test Case 141 Input:
1111 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 11 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
Output:
0
Test Case 142 Input:
111 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 11 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
Output:
0
Test Case 143 Input:
11a1
Output:
0
Test Case 144 Input:
11null
Output:
0
Test Case 145 Input:
11 2
Output:
0
Test Case 146 Input:
12
Output:
0
Test Case 147 Input:
12345
Output:
0
Test Case 148 Input:
1234567893123
Output:
0
Test Case 149 Input:
123456789512345
Output:
0
Test Case 150 Input:
1ABC
Output:
0
Test Case 151 Input:
1Test Case Input:21 2Test Case Input:31 22 3Test Case Input:41 22 33 4Test Case Input:51 21 33 43 5
Output:
0
Test Case 152 Input:
1a b c d e f g h i j k l m n o p q r s t u v w x y z1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 153 Input:
1three
Output:
0
Test Case 154 Input:
1 2 34 5 67 8 901 2 3
Output:
0
Test Case 155 Input:
1 2 34 5 67 8 93-1 0 1
Output:
0
Test Case 156 Input:
1 2 34 5 67 8 930 0 0
Output:
0
Test Case 157 Input:
1 2 34 5 67 8 930 1 2
Output:
0
Test Case 158 Input:
1 2 34 5 67 8 931 0 3
Output:
0
Test Case 159 Input:
1 2 34 5 67 8 931 2 3
Output:
0
Test Case 160 Input:
1 2 34 5 67 8 931 5 9
Output:
0
Test Case 161 Input:
1 2 34 5 67 8 9310 11 12
Output:
0
Test Case 162 Input:
1 2 34 5 67 8 9310 A 12
Output:
0
Test Case 163 Input:
1 2 34 5 67 8 93A B C
Output:
0
Test Case 164 Input:
1 2 34 5 67 8 93a b c
Output:
0
Test Case 165 Input:
1 2 34 5 67 8 9A
Output:
0
Test Case 166 Input:
1 2 34 5 67 8 9ATest Case Input:1 2 34 5 67 8 90Test Case Input:1 2 34 5 67 8 910Test Case Input:1 2 34 5 67 8 9-5Test Case Input:1 2 34 5 67 8 92 5 7 10Test Case Input:1 2 34 5 67 8 9-1 -2 -3Test Case Input:1 2 34 5 67 8 93 5 2Test Case Input:1 2 34 5 67 8 91 2 3 4
Output:
0
Test Case 167 Input:
1 2 34 5 67 8 9ATest Case Input:1 2 34 5 67 8 90Test Case Input:1 2 34 5 67 8 93Test Case Input:1 2 34 5 67 8 91.5Test Case Input:1 2 34 5 67 8 910
Output:
0
Test Case 168 Input:
1 2 34 5 67 8 9ATest Case Output:NumberFormatException
Output:
0
Test Case 169 Input:
1 2 34 5 67 8 9A1 2 3
Output:
0
Test Case 170 Input:
1 2 34 5 A7 8 926 7
Output:
0
Test Case 171 Input:
1 A 34 5 67 8 931 2 3
Output:
0
Test Case 172 Input:
1 A 34 B 67 C 931 2 3
Output:
0
Test Case 173 Input:
1.512345678910
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 174 Input:
1.52.53.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 175 Input:
1.5 2.5 3.54.5 5.5 6.57.5 8.5 9.531.5 2.5 3.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 176 Input:
10
Output:
10
Test Case 177 Input:
10"abc"
Output:
10
Test Case 178 Input:
100
Output:
10
Test Case 179 Input:
1001111111111
Output:
10
Test Case 180 Input:
101-2345678910
Output:
10
Test Case 181 Input:
1012345678910
Output:
10
Test Case 182 Input:
101 21 32 42 53 63 74 84 95 10
Output:
10
Test Case 183 Input:
101 22 32 43 53 64 74 85 96 10
Output:
10
Test Case 184 Input:
102
Output:
10
Test Case 185 Input:
103
Output:
10
Test Case 186 Input:
1053
Output:
10
Test Case 187 Input:
1053PREVIOUSLY GENERATED TEST CASES
Output:
10
Test Case 188 Input:
1057PREVIOUSLY GENERATED TEST CASES
Output:
10
Test Case 189 Input:
10abc
Output:
10
Test Case 190 Input:
10$0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 191 Input:
10-0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 192 Input:
10.10101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 193 Input:
10.5153.2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 194 Input:
10.515.73.2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 195 Input:
10.52.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 196 Input:
10/0
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 197 Input:
10/0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 198 Input:
100200-50
Output:
100
Test Case 199 Input:
1000
Output:
2000
Test Case 200 Input:
1000000
Output:
2000000
Test Case 201 Input:
100000000112345678910
Output:
2000000000
Test Case 202 Input:
100000000212345678910
Output:
2000000000
Test Case 203 Input:
1005
Output:
2005
Test Case 204 Input:
10101010
Output:
20202010
Test Case 205 Input:
1010101010
Output:
2020202010
Test Case 206 Input:
10101010101a
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 207 Input:
1010101010a
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 208 Input:
1011100110
Output:
2022200110
Test Case 209 Input:
10a01010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 210 Input:
10a0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 211 Input:
10a0101010a
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 212 Input:
10a01010a
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 213 Input:
10a0a01010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 214 Input:
10a0b1010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 215 Input:
10a10a1010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 216 Input:
11-1-2-3-4-5-6-7-8-9-10-11
Output:
10
Test Case 217 Input:
1100110011
Output:
-2094747286
Test Case 218 Input:
1110001010
Output:
-2074965286
Test Case 219 Input:
12.34
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 220 Input:
12/0
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 221 Input:
12345
Output:
24345
Test Case 222 Input:
1234567890
Output:
-1825831906
Test Case 223 Input:
1500
Output:
3000
Test Case 224 Input:
1a0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 225 Input:
1a01101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 226 Input:
1a0b1c0d1e
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 227 Input:
1a0b1c0d1e0f
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 228 Input:
1a10b10101
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 229 Input:
1a2b3c4d5e6f7g8h9i0j
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 230 Input:
21-1
Output:
0
Test Case 231 Input:
21 2
Output:
0
Test Case 232 Input:
21 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 233 Input:
22-10
Output:
0
Test Case 234 Input:
23-10
Output:
0
Test Case 235 Input:
23-202
Output:
0
Test Case 236 Input:
23 5 7 9 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 41 43 45 47 49 51 532 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 237 Input:
24 2 ABC
Output:
0
Test Case 238 Input:
24 2.5
Output:
0
Test Case 239 Input:
24 2a
Output:
0
Test Case 240 Input:
24 ABC
Output:
0
Test Case 241 Input:
25 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 85 90 95 100 105 110 115 120 125 130 1351 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26PREVIOUSLY GENERATED TEST CASES
Output:
0
Test Case 242 Input:
2.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 243 Input:
2.5103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 244 Input:
2.53.74.2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 245 Input:
30-202
Output:
0
Test Case 246 Input:
31-23
Output:
0
Test Case 247 Input:
312"3"
Output:
0
Test Case 248 Input:
312"three"
Output:
0
Test Case 249 Input:
312-3
Output:
0
Test Case 250 Input:
3123
Output:
0
Test Case 251 Input:
312three
Output:
0
Test Case 252 Input:
31 21 3
Output:
0
Test Case 253 Input:
31 21 3Test Case Output:Fennec
Output:
0
Test Case 254 Input:
31 22 3
Output:
0
Test Case 255 Input:
31 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 256 Input:
32
Output:
0
Test Case 257 Input:
32"abc"
Output:
0
Test Case 258 Input:
32-120
Output:
0
Test Case 259 Input:
32-202
Output:
0
Test Case 260 Input:
321 -2 3
Output:
0
Test Case 261 Input:
32101
Output:
0
Test Case 262 Input:
32468
Output:
0
Test Case 263 Input:
32 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 264 Input:
33101
Output:
0
Test Case 265 Input:
34
Output:
0
Test Case 266 Input:
34123
Output:
0
Test Case 267 Input:
34 -2 7
Output:
0
Test Case 268 Input:
34 2 "ABC"
Output:
0
Test Case 269 Input:
34 2 -7
Output:
0
Test Case 270 Input:
34 2 -7 1 9
Output:
0
Test Case 271 Input:
34 2 7
Output:
0
Test Case 272 Input:
34 2 7 1
Output:
0
Test Case 273 Input:
34 2 7 1 -9
Output:
0
Test Case 274 Input:
34 2 7 1 9
Output:
0
Test Case 275 Input:
34 2 7 1 ABC
Output:
0
Test Case 276 Input:
34 2 7 ABC
Output:
0
Test Case 277 Input:
34 2 ABC
Output:
0
Test Case 278 Input:
34 2 abc
Output:
0
Test Case 279 Input:
34 ABC 7
Output:
0
Test Case 280 Input:
39 8 7
Output:
0
Test Case 281 Input:
3a b c d e f g h i j k l m n o p q r s t u v w x y z1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
0
Test Case 282 Input:
3.14
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 283 Input:
3.54 2 ABC
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 284 Input:
41234
Output:
0
Test Case 285 Input:
4123four
Output:
0
Test Case 286 Input:
41 21 33 4
Output:
0
Test Case 287 Input:
41 22 33 4
Output:
0
Test Case 288 Input:
41 23 4
Output:
0
Test Case 289 Input:
42-305
Output:
0
Test Case 290 Input:
44 2 -7 ABC
Output:
0
Test Case 291 Input:
44 2 7 ABC
Output:
0
Test Case 292 Input:
44 2 ABC
Output:
0
Test Case 293 Input:
5
Output:
5
Test Case 294 Input:
503
Output:
5
Test Case 295 Input:
50abc123
Output:
5
Test Case 296 Input:
50 11 22 33 4
Output:
5
Test Case 297 Input:
5111110
Output:
5
Test Case 298 Input:
51234"five"
Output:
5
Test Case 299 Input:
512345
Output:
5
Test Case 300 Input:
51234five
Output:
5
Test Case 301 Input:
513254
Output:
5
Test Case 302 Input:
51 21 32 42 5
Output:
5
Test Case 303 Input:
51 21 32 43 5
Output:
5
Test Case 304 Input:
51 21 32 44 5
Output:
5
Test Case 305 Input:
51 21 33 43 5
Output:
5
Test Case 306 Input:
51 21 33 43 51 6(Note: This test case is designed to raise an ArrayIndexOutOfBoundsException as the program tries to access index 6 of the array, which is out of bounds.)
Output:
5
Test Case 307 Input:
51 21 33 43 55 6
Output:
5
Test Case 308 Input:
51 21 34 5Test Case Input:41 22 33 4Test Case Input:31 22 3Test Case Input:0Test Case Input:-51 21 33 43 5Test Case Input:41 21 33 4Test Case Input:71 21 32 42 53 63 7Test Case Input:31 22 3Test Case Input:-31 22 3Test Case Input:31 22 3
Output:
5
Test Case 309 Input:
51 22 32 44 5
Output:
5
Test Case 310 Input:
51 22 33 44 5
Output:
5
Test Case 311 Input:
51 2 3 4 5
Output:
5
Test Case 312 Input:
51 2 3 4 51 2 3 4 51 2 3 4 51 2 3 4 51 2 3 4 5
Output:
5
Test Case 313 Input:
51 2 3 4 51 2 3 4 55 4 3 2 12 3 4 5 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 1
Output:
5
Test Case 314 Input:
51 2 3 4 51 2 3 4 56 7 8 9 1011 12 13 14 1516 17 18 19 2021 22 23 24 25
Output:
5
Test Case 315 Input:
510-3
Output:
5
Test Case 316 Input:
5103
Output:
5
Test Case 317 Input:
52
Output:
5
Test Case 318 Input:
52-2042
Output:
5
Test Case 319 Input:
52-5
Output:
5
Test Case 320 Input:
5210101
Output:
5
Test Case 321 Input:
5210110
Output:
5
Test Case 322 Input:
5211001
Output:
5
Test Case 323 Input:
5211010
Output:
5
Test Case 324 Input:
5211111
Output:
5
Test Case 325 Input:
5211a01
Output:
5
Test Case 326 Input:
5211abc
Output:
5
Test Case 327 Input:
5212abc
Output:
5
Test Case 328 Input:
52abc123
Output:
5
Test Case 329 Input:
52null
Output:
5
Test Case 330 Input:
52xyz123
Output:
5
Test Case 331 Input:
52 4 3 1 5
Output:
5
Test Case 332 Input:
53-2-31-54
Output:
5
Test Case 333 Input:
53-240-13
Output:
5
Test Case 334 Input:
53-240-31
Output:
5
Test Case 335 Input:
531-23-40
Output:
5
Test Case 336 Input:
531-23-45
Output:
5
Test Case 337 Input:
531 -2 3 -4 5
Output:
5
Test Case 338 Input:
5310011
Output:
5
Test Case 339 Input:
532
Output:
5
Test Case 340 Input:
53 1 4 2 5
Output:
5
Test Case 341 Input:
5472119
Output:
5
Test Case 342 Input:
54 -2 7 1 9
Output:
5
Test Case 343 Input:
54 1 3 2 5
Output:
5
Test Case 344 Input:
54 2 6 1 3
Output:
5
Test Case 345 Input:
54 2 7 1 "9"
Output:
5
Test Case 346 Input:
54 2 7 1 "ABC"
Output:
5
Test Case 347 Input:
54 2 7 1 -2147483648
Output:
5
Test Case 348 Input:
54 2 7 1 -9
Output:
5
Test Case 349 Input:
54 2 7 1 -9.5
Output:
5
Test Case 350 Input:
54 2 7 1 6
Output:
5
Test Case 351 Input:
54 2 7 1 9
Output:
5
Test Case 352 Input:
54 2 7 1 9 A
Output:
5
Test Case 353 Input:
54 2 7 1 9.5
Output:
5
Test Case 354 Input:
54 2 7 1 ABC
Output:
5
Test Case 355 Input:
54 2 7 1 XYZ
Output:
5
Test Case 356 Input:
54 2 7 1 null
Output:
5
Test Case 357 Input:
54 3 2 1 0
Output:
5
Test Case 358 Input:
54 a 7 1 9
Output:
5
Test Case 359 Input:
554321
Output:
5
Test Case 360 Input:
55 1 3 2 4
Output:
5
Test Case 361 Input:
55 4 3 2 1
Output:
5
Test Case 362 Input:
59 3 7 1 5
Output:
5
Test Case 363 Input:
5.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 364 Input:
5.5103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 365 Input:
5.510.23.7
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 366 Input:
500.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 367 Input:
61 21 31 42 52 6
Output:
5
Test Case 368 Input:
61 21 32 42 53 6
Output:
5
Test Case 369 Input:
61 21 32 42 54 6
Output:
5
Test Case 370 Input:
61 22 31 44 55 6
Output:
5
Test Case 371 Input:
61 22 32 43 54 6
Output:
5
Test Case 372 Input:
62
Output:
5
Test Case 373 Input:
71 21 32 42 53 63 7
Output:
5
Test Case 374 Input:
73
Output:
5
Test Case 375 Input:
81 21 32 42 53 63 74 8Explanation: This test case creates a scenario where the input graph has 8 vertices and 7 edges.
Output:
5
Test Case 376 Input:
9a8b7c6d5e
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 377 Input:
A B CD E FG H I3J K L
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 378 Input:
Hello World
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 379 Input:
a123
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 380 Input:
a2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 381 Input:
a211001
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 382 Input:
a4 2 7 1 9
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 383 Input:
a 2 34 5 67 8 931 2 3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 384 Input:
a b cd e fg h i2j k l
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 385 Input:
a b cd e fg h i31 2 3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 386 Input:
a b cd e fg h i3j k l
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 387 Input:
a b cd e fg h i3x y z
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 388 Input:
a010101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 389 Input:
ab0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 390 Input:
abc
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 391 Input:
abcTest Case Output:InputMismatchException
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 392 Input:
abc12345678910
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 393 Input:
abc103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 394 Input:
abc2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 395 Input:
abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abcTest Case Input:abc
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 396 Input:
abc0101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 397 Input:
abc1010101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 398 Input:
abc12345
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 399 Input:
abcd1010101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 400 Input:
abcde1010101010
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 401 Input:
abcde12345
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 402 Input:
abcdefg
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 403 Input:
null
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 404 Input:
null103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 405 Input:
null2 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 406 Input:
null101010101
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 407 Input:
true
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 408 Input:
true103
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 409 Input:
xyz
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 410 Input:
"abc" "2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 411 Input:
34 2 ABC
Output:
0
